src/simplnx/Utilities/Math/GeometryMath.hpp

```diff
@@ -605,7 +759,8 @@ char IsPointInPolyhedron(const nx::core::TriangleGeom& faces, const std::vector<
   } /* End while loop */
 
   /* q strictly interior to polyhedron if an odd number of crossings. */
-  if((crossings % 2) == 1)
+  // Bitwise & for speed
+  if(crossings & 1)
```

Comment:
Since this is a common optimization, the compiler can generally figure out when to apply it. In this link, the compiler is able to optimize all three to same assembly for unsigned integers. Then it differs on `value % 2 == 1` for signed integers. `value % 2 != 0` gives the same assembly for signed and unsigned. I tend to think that `value % 2 != 0` is the most readable while also giving the same assembly. Alternatively we could put it in a macro like `#define IS_ODD(value) (value % 2 != 0)`. In any case this is only a minor note and not a big deal either way.

https://godbolt.org/z/7KWsEYzex